asciidoc/runtime/src/main/java/io/quarkiverse/qute/web/asciidoc/runtime/AsciidocSectionHelperFactory.java

```diff
@@ -29,8 +23,8 @@ public AsciidocSectionHelper initialize(SectionInitContext context) {
     }
 
     @TemplateExtension(matchNames = { "asciidocify", "asciidocToHtml" })
-    static String convertToAsciidoc(String text, String ignoredName) {
-        return CONVERTER.apply(text);
+    static RawString convertToAsciidoc(String text, String ignoredName) {
```

Comment:
How does it fix the problem with section params in #146? 

IIUIC this merely changes the behavior of `{text.asciidocToHtml}` in a way that it corresponds to `{text.asciidocToHtml.raw}`. Which is probably ok, unless someone wants to escape the result of conversion... which probably does not make a lot of sense, right?

Reply:
What I was trying to achieve with an attribute was to prevent the the rendered content of beeing escaped. So if we have another to achieve the same it's fine. 
It may be still be of some use to be able to pass attributes to the section but this would be a good start.

Reply:
> How does it fix the problem with section params in #146?
> 
> IIUIC this merely changes the behavior of `{text.asciidocToHtml}` in a way that it corresponds to `{text.asciidocToHtml.raw}`. Which is probably ok, unless someone wants to escape the result of conversion... which probably does not make a lot of sense, right?

Yes that's why we make it the default behaviour, it doesn't make sense to use it escaped..

Reply:
> Which is probably ok, unless someone wants to escape the result of conversion... 

Actually, in that case a user can do something like `{text.asciidocToHtml.value}`.